contracts/Strategy.sol

```diff
@@ -22,34 +24,33 @@ interface IUni {
         returns (uint256[] memory amounts);
 }
 
-// Import interfaces for many popular DeFi projects, or add your own!
-//import "../interfaces/<protocol>/<Interface>.sol";
-
-contract Strategy is BaseStrategy {
+contract Strategy is BaseStrategy, Synthetix {
     using SafeERC20 for IERC20;
     using Address for address;
     using SafeMath for uint256;
 
-    ICurveFi public curvePool; // =  ICurveFi(address(0x4CA9b3063Ec5866A4B82E437059D2C43d1be596F));
-    ICrvV3 public curveToken; // = ICrvV3(address(0xb19059ebb43466C323583928285a49f558E572Fd));
+    ICurveFi public curvePool;
+    ICrvV3 public curveToken;
+
+    uint256 public susdBuffer = 1_000; // 10% (over 10_000 BPS) amount of sUSD that should not be exchanged for sETH
```

Comment:
You are already setting this value in initialize. remove?

Reply:
done